# alpine_msgs/msg/AlpineDualImu.msg
uint32 epoch_ms
float32[11] imu1
float32[11] imu2
